libs/SalesforceSDK/src/com/salesforce/androidsdk/developer/support/activities/ShowDeveloperSupportActivity.kt

```diff
@@ -0,0 +1,19 @@
+package com.salesforce.androidsdk.developer.support.activities
+
+import androidx.appcompat.app.AppCompatActivity
+
+/**
+ * An Activity that, when started, finishes immediately and shows the developer
+ * support dialog from the previous Activity.  SalesforceSDKManager implements
+ * the logic for observing the Activity lifecycle and showing the developer
+ * support dialog.
+ */
+class ShowDeveloperSupportActivity : AppCompatActivity() {
```

Comment:
This Activity serves as the "action" for the new notification.  I had tried the old broadcast intent trick, but then remembered how newer API levels require a call to `Activity.startActivity` from notification actions.  Finishing this activity accomplishes the same integration in the `SalesforceSDKManager` Activity Lifecycle Callback.